src/OrchardCore/OrchardCore.ContentManagement.Display/ContentItemDisplayManager.cs

```diff
@@ -64,10 +64,17 @@ public async Task<IShape> BuildDisplayAsync(ContentItem contentItem, IUpdateMode
             }
 
             var stereotype = contentTypeDefinition.GetSettings<ContentTypeSettings>().Stereotype;
-            var actualDisplayType = string.IsNullOrEmpty(displayType) ? "Detail" : displayType;
-            var actualShapeType = stereotype ?? "Content";
+            var actualDisplayType = String.IsNullOrEmpty(displayType) ? "Detail" : displayType;
+            var hasStereotype = !String.IsNullOrWhiteSpace(stereotype);
```

Comment:
string

Reply:
String is fine. With global using we will surely eventually have `using System`.